platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/bfn_extensions/psu_sensors.py

```diff
@@ -0,0 +1,125 @@
+from netifaces import ifaddresses, AF_INET6
+from subprocess import Popen, PIPE, DEVNULL
+import json
+import os
+
+psu_drivers = {
```

Comment:
Is it the same for Newport/Montara/Maveriks?
Is it the same for all releases of the boards?

Reply:
refactored to using "PSU%d % index" format